src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
+
+interface GeneralCellStyle {
+    borderStyle: "solid" | "dashed" | "dotted" | undefined;
+    borderWidth: number;
+    borderLeftWidth: number;
+    borderTopWidth: number;
+    padding: string;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const generalCellStyle: GeneralCellStyle = {
+    borderStyle: "solid",
+    borderWidth: 1,
+    borderLeftWidth: 0,
+    borderTopWidth: 0,
+    padding: "3px",
+};
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "12px",
+    },
+    card: {
+        margin: "0px 40px 20px",
+    },
+    table: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderRightWidth: 0,
+        borderBottomWidth: 0,
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "30px",
+    },
+    dateLocationWrap: {
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        marginBottom: "5px",
+    },
+    svg: {
+        margin: "0 1px",
+        padding: "0 1px",
+    },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
+
+    // headers
+    header_flag: { flex: 1, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_name: { flex: 4, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_postcode: { flex: 4, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_time: { flex: 2, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_collection: { flex: 4, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_six: { flex: 10, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+
+    // cells
+    cell_logo: { flex: 1, ...generalCellStyle },
+    cell_name: { flex: 4, ...generalCellStyle },
+    cell_postcode: { flex: 4, ...generalCellStyle },
+    cell_time: { flex: 2, ...generalCellStyle },
+    cell_collection: { flex: 4, ...generalCellStyle },
+    cell_instructions: { flex: 10, ...generalCellStyle },
+});
+
+const dateTimeToAMPM = (datetime: string): string => {
+    return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
+};
+
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
+    <Svg width={12} height={12} viewBox="0 0 512 512" style={styles.svg}>
+        <Path
+            fill={fill ? color : undefined}
+            stroke={color}
+            strokeWidth={40}
+            d={icon.icon[4] as string}
+        />
+        {/*IconDefinition[4] is defined as string | string[] but in reality there is no use case*/}
+        {/*with string[] as svgPathData*/}
+    </Svg>
+);
+
+const DayOverviewerDate: React.FC<{ date: Date }> = ({ date }) => {
+    const dateString = date.toLocaleDateString("sv-SE");
```

Comment:
is this Swedish? (and is this meant to be there?)

Reply:
Not sure what language is it
But the format looks good as I want something like `2023-08-15` and this returns it

Reply:
In that case should we just manually do it via, getDate, getFullYear and concat it manually? sv-SE is not clear for other developers and could cause problems later on. I agree that toLocalDateString is more concise but might not fit the use case sometimes.

Reply:
Think you should be able to change this to something like gb-EN or something? Seems wrong to have swedish in there (also might end up being a problem w time zones!)